src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,61 @@
+.calendar {
+  max-width: 7 * $cell-size + 7 * $cell-gap + 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  margin: 50px auto;
```

Comment:
Position can be only on element, not block.

Reply:
agree
Just put on body 
```
body {
   display: flex;
   justify-content: center;
   align-items: center;
   height: 100vh;
}
```